examples/player/simid_player.js

```diff
@@ -115,10 +115,19 @@ class SimidPlayer {
    * a session.
    */
   initializeAd() {
+
+    if (!this.isLinearAd_ && !this.isValidDimensions_(this.getCreativeDimensions_())) {
+      console.log('Dimensions bigger than player');
+      return;
+    }
+
+    // After the iframe is created the player will wait until the ad
```

Comment:
minor nit but should this comment be in `/** */ ` form?

Reply:
Good thought, but we usually reserve `/** */` comments for jsdocs. Inline comments like this are fine to use the `//` syntax, or `/* */` if they're going to be very long (notice the jsdoc starts with 2 stars, a regular inline mult-line comment uses 1)